libs/coin-modules/coin-filecoin/src/types/common.ts

```diff
@@ -6,12 +6,12 @@ export enum TxStatus {
 }
 
 export interface EstimatedFeesRequest {
-  to?: string;
+  to?: string | undefined;
   from: string;
   methodNum?: number;
   blockIncl?: number;
-  params?: string;
-  value?: string;
+  params?: string | undefined;
+  value?: string | undefined;
```

Comment:
Do we want to add `undefined` as an option? Is optional not enough to handle the fact we don't want to pass the property?